tests/e2e/vmgroupmisc_test.go

```diff
@@ -224,3 +225,37 @@ func (s *vgBasicSuite) TestDSAccessPrivilegeForUserVG(c *C) {
 
 	misc.LogTestEnd(c.TestName())
 }
+
+// Test steps:
+// 1. Create a vm VM1 using govc
+// 2. Create a vmgroup and associate VM1 to it.
+// 2. Execute vmgroup ls command.
+// 3. Delete the vm that was added to the vmgroup.
+// 4. Again execute vmgroup ls command to verify command works fine.
+func (s *vgBasicSuite) TestDeleteVMFromVmgroup(c *C) {
+	misc.LogTestStart(c.TestName())
+	vmName := "VM_" + inputparams.GetRandomNumber()
```

Comment:
Why do you want to create a new vmgroup here? Why not using existing vmgroup "vmGroupName"?